api/serializers.py

```diff
@@ -235,4 +235,4 @@ def create(self, validated_data):
 
 class LectureSummarySerializer(serializers.Serializer):
     summary = serializers.CharField(required=True)
-    recording_id = serializers.IntegerField(required=True)
+    recording_id_id = serializers.IntegerField(required=False)  # corrected column name
```

Comment:
Changing it to recording_id and adding the source argument breaks the endpoint. I think it's best for now to keep it as recording_id_id since that is the name of the column in the table.

Reply:
I managed to resolve this. It should appear as recording_id now.